### PR TITLE
add Einstein Probe mission page

### PR DESCRIPTION
**Pull request by jracusin**
GCN - Missions - Einstein Probe

### CODE REVIEW

**Comment by Vidushi-GitHub** [comment]
[@modot](https://github.com/modot) This PR is for EP mission page, which was discussed earlier as well. Could you provide information for us to include on latency?

File: app/routes/missions.einsteinprobe/route.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+</div>
+
+**JSON-Serialized GCN Notices Types in GCN Kafka:**
+
+<div className="overflow-table">
+
+| Type                                   | Contents                               | Latency |
+| -------------------------------------- | -------------------------------------- | ------- |
+| `gcn.notices.einstein_probe.wxt.alert` | Localization, Count Rate, Significance | TBD     |
```
Comment (modot) [inline app/routes/missions.einsteinprobe/route.mdx:40]: Latency should be ~1 min

File: app/routes/missions.einsteinprobe/route.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+</div>
+
+**Yearly Trigger Rates:**
+
+<div className="overflow-table">
+
+| Instrument | Type                                                   | Rates |
+| ---------- | ------------------------------------------------------ | ----- |
+| WXT        | GRBs, other transients, and non-astrophysical triggers | TBD   |
```
Comment (modot) [inline app/routes/missions.einsteinprobe/route.mdx:50]: Rates should be ~100

File: app/routes/missions.einsteinprobe/route.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+<div className="overflow-table">
+
+| Instruments                | Energy Range | Field of View        | Localization           |
+| -------------------------- | ------------ | -------------------- | ---------------------- |
+| Wide Field X-ray Telescope | 0.5 – 5 keV  | 3600 deg<sup>2</sup> | 5 arcmin (FWHM)        |
```
Comment (modot) [inline app/routes/missions.einsteinprobe/route.mdx:29]: Energy Range should be 0.5 - 4 keV
Localization should be 2 arcmin (FWHM)

**Review by modot — CHANGES_REQUESTED** [review]
Please see the comments I made recently.

File: app/routes/missions.einsteinprobe/route.mdx (hunk 3 above)
Comment (Vidushi-GitHub) [inline app/routes/missions.einsteinprobe/route.mdx:29]:
```suggestion
| Wide Field X-ray Telescope | 0.5 - 4 keV  | 3600 deg<sup>2</sup> | 2 arcmin (FWHM)        |
```

File: app/routes/missions.einsteinprobe/route.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+</div>
+
+**JSON-Serialized GCN Notices Types in GCN Kafka:**
+
+<div className="overflow-table">
+
+| Type                                   | Contents                               | Latency |
+| -------------------------------------- | -------------------------------------- | ------- |
+| `gcn.notices.einstein_probe.wxt.alert` | Localization, Count Rate, Significance | TBD     |
```
Comment (Vidushi-GitHub) [inline app/routes/missions.einsteinprobe/route.mdx:40]:
```suggestion
| `gcn.notices.einstein_probe.wxt.alert` | Localization, Count Rate, Significance | ~1 min    |
```

File: app/routes/missions.einsteinprobe/route.mdx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+</div>
+
+**Yearly Trigger Rates:**
+
+<div className="overflow-table">
+
+| Instrument | Type                                                   | Rates |
+| ---------- | ------------------------------------------------------ | ----- |
+| WXT        | GRBs, other transients, and non-astrophysical triggers | TBD   |
```
Comment (Vidushi-GitHub) [inline app/routes/missions.einsteinprobe/route.mdx:50]:
```suggestion
| WXT        | GRBs, other transients, and non-astrophysical triggers | ~100   |
```

**Comment by Vidushi-GitHub** [comment]
@modot  Could you please confirm the final modifications? see the screenshot of mission page:
Screenshot 2024-04-11 at 1 23 51 AM

**Review by Vidushi-GitHub — APPROVED** [review]

**Comment by modot** [comment]
@Vidushi-GitHub Could you please remove `(FWHM)` for the WXT localization in the first sheet? `2 arcmin` is sufficient.

File: app/routes/missions.einsteinprobe/route.mdx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+<div className="overflow-table">
+
+| Instruments                | Energy Range | Field of View        | Localization           |
+| -------------------------- | ------------ | -------------------- | ---------------------- |
+| Wide Field X-ray Telescope | 0.5 - 4 keV  | 3600 deg<sup>2</sup> | 2 arcmin (FWHM)        |
```
Comment (Vidushi-GitHub) [inline app/routes/missions.einsteinprobe/route.mdx:29]:
```suggestion
| Wide Field X-ray Telescope | 0.5 - 4 keV  | 3600 deg<sup>2</sup> | 2 arcmin        |
```
@modot done!

**Comment by Vidushi-GitHub** [comment]
> @Vidushi-GitHub Could you please remove `(FWHM)` for the WXT localization in the first sheet? `2 arcmin` is sufficient.

@modot Corrected! Any other suggestion?

**Comment by modot** [comment]
@Vidushi-GitHub All good now. Thanks!

**Review by modot — APPROVED** [review]
All good. Thanks!

**Comment by Vidushi-GitHub** [comment]
@jracusin and @lpsinger all feedbacks from the team has been implemented, shall we merge it?

File: app/routes/missions.einsteinprobe/logo.png
Comment (lpsinger) [inline app/routes/missions.einsteinprobe/logo.png:1]: Please obtain an SVG version of this image.

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Comment by Vidushi-GitHub** [comment]
@modot Please attach the SVG image of the EP logo here (at GitHub). Thanks

File: app/routes/missions.einsteinprobe/route.mdx [hunk 7]
```diff
@@ -0,0 +1,52 @@
+---
+handle:
+  breadcrumb: Einstein Probe
+---
+
+import logo from './logo.png'
```
Comment (Vidushi-GitHub) [inline app/routes/missions.einsteinprobe/route.mdx:6]:
```suggestion
import logo from './logo.svg'
```

**Comment by Vidushi-GitHub** [comment]
@jracusin EP mission page is almost done. 

Things needed for your branch:
-git pull this branch for updating the changes in your branch
-Please remove logo.png, as logo.svg is added
-Rebase and merge

**Comment by jracusin** [comment]
@Vidushi-GitHub This branch has been rebased.

**Comment by Vidushi-GitHub** [comment]
@modot, @jracusin and @lpsinger, Kindly approve the final changes. 
Screenshot 2024-04-15 at 12 30 47 PM

**Review by modot — APPROVED** [review]
Great! Thanks a lot for all the effort!

**Review by Vidushi-GitHub — APPROVED** [review]

**Pull request merged**